strawberryfields/gbs/points.py

```diff
@@ -26,7 +26,7 @@
 ---------------
 
 A point process is a mechanism that randomly generates points among a set of possible outcomes.
```

Comment:
I think this first sentence can be slightly modified. Something such as "A point process is a mechanism that generates random point patterns among a set of possible outcomes."